RFC.md

```diff
@@ -124,18 +136,22 @@ The Tool Request Schema is designed to encapsulate the details of a tool invocat
 
 - **Run and Execution Identification:**
 
+[//]: # (Could we have the run_id be optional and if not provided, the server generates one?)
+[//]: # (It's not clear why two required identifiers are needed.)
   - **`run_id`**: Globally unique identifier for the overall run.
   - **`execution_id`**: Unique identifier for the specific tool execution.
-  - **`created_at`**: Timestamp indicating when the request was created.
 
 - **Tool Metadata:**
 
+[//]: # (This looks like a unique identifier for the tool. Perhaps we could rename as `id` or `tool_id`?)
   - **`tool`**: Contains the tool's name, the toolkit to which it belongs, and the toolkit version.
 
 - **Input Parameters:**
 
   - **`inputs`**: An object containing the parameters needed by the tool. This field supports additional properties to accommodate various tool-specific inputs.
 
+[//]: # (Could you document more about how the context is used? Why the different parts are needed?)
+[//]: # (In the event that the execution of the tool is hosted on the same server as the registry, why is any context needed?)
```

Comment:
I'll add more detail here. `context` is optional and only relevant for tools that require information that isn't strictly a parameter (like an API key or OAuth token)